utils/zsh/patches/020-replace-missing-autotools-ln-with-ln_s.patch

```diff
@@ -0,0 +1,21 @@
+--- a/Src/Makefile.in
```

Comment:
dtto, but this one in my point of view should be in separate commit

Reply:
Then it should be in a commit before this, as this doesn't build without. And then this PR would become incompatible, as this is the patching order, that it's applied afterwards pcre2 patch.

Are you aware that byrocracy that you put me through is larger than making things work the way you wanted them to work? Just do concrete proposals on everything and I'll accept them all; after all, this is your project to update to completely to libpcre2 and I just volunteered with something like I've done on openwrt project for 20 years since my first wrt54gs gen1 which I still have 5 pieces in my attic... I do realise newer rules, but for someone like me it's a bit difficult since when patchwork was the way to go, it was already almost a reason to resign from contributing due to it becoming so difficult and trouble-some. Even though I talk (or write) a lot, I've always been more on the action and little less on the conversation side. There was a time when making things happen was the most trouble-some part and there was gratitude on what was given; these days I update software I maintain with 3 month cycles, since otherwise I'm writing changelogs and comments to every commit for all my free time, which more or less seems only purposeful slowing of things, because those changelogs and comments become obsolete on next update; just like this patch does. On next update, it is out dated and obsolete, won't even be possible to apply it, as it's already applied in mainstream code.

I'm currently working on another project(not openwrt related) and as tomorrow I need to go to school to update my licenses, I'm going to be very busy for this weekend; so, provide concrete examples please and I'll apply them. I talk a lot, but here's my weakest spot: documentation. Second weak spot is persuasion. Thank you.